exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
```

Comment:
Great work Kwanda.
What do you think would happen if we dropped all the else statements in the functions below ?
```suggestion
function isNegative(num) {
  if (num < 0 ) {
     return true;
  }
  return false;
```